magento/magento2
# Fix typo in SQL join when joining custom option prices for price indexer

### Description (*)
In case where product has price variation for custom option per website, only default value (`store_id = 0`) is taken into an account due to typo in SQL statement.


### Fixed Issues (if relevant)

1. N/A

### Manual testing scenarios (*)

This one is pretty obvious from diff, but...
1. Create  scenario where simple product price is regulated per website and have at least two websites
2. Set default price to amount X
3. Set price of custom option (on default view) to amount to Y
4. Switch scope to another website and set price of custom option to be 0
5. Execute price indexer
6. Difference between min and max price should be visible only on one website, however all websites will have this difference

### Contribution checklist (*)
